website/content/api-docs/operator/license.mdx

```diff
@@ -72,134 +72,4 @@ $ curl \
   },
   "Warnings": []
 }
-```
-
-## Updating the Consul License
-
-This endpoint updates the Consul license and returns some of the
-license contents as well as any warning messages regarding its validity.
-
-| Method | Path                | Produces           |
-| ------ | ------------------- | ------------------ |
-| `PUT`  | `/operator/license` | `application/json` |
```

Comment:
Given that we don't have versioned docs, should we just label the create, update, and delete APIs as deprecated for Consul 1.10? Users on prior versions may still need to reference these docs.

Reply:
Probably. I can make the updates.

Reply:
I went ahead and added big deprecation notices to the two HTTP endpoints as well as the CLI commands instead of just removing them. We really need versioned docs.